package-lock.json

```diff
@@ -5,6 +5,9 @@
   "packages": {
     "": {
       "hasInstallScript": true,
+      "dependencies": {
```

Comment:
I believe that the changes to the package-lock.json can be reverted.

Reply:
ah thanks for catching this

reverted w/ https://github.com/Qiskit/qiskit.org/pull/3019/commits/8a837d362d70de7a1944495fd27ae3c1f2ea715f